src/material/core/ripple/ripple.ts

```diff
@@ -91,6 +91,9 @@ export class MatRipple implements OnInit, OnDestroy, RippleTarget {
   @Input('matRippleDisabled')
   get disabled() { return this._disabled; }
   set disabled(value: boolean) {
+    if (value) {
+      this.fadeOutAll();
```

Comment:
I'm unsure if this is what we want. I could imagine there being persistent ripples (like focus indicators) which should not necessarily fade out if `disabled` is set to `true`. Would it make sense to only fade out non-persistent ripples?

Reply:
Only fade out non-persistent ripples sounds good to me. Should I create a new `fadeOutAllPersistent()` function in `ripple-renderer`? It seems like the current method fades out all ripples.

Reply:
Yeah, I'm unsure about the naming here, but a new method in the ripple renderer sounds reasonable. Maybe something around `fadeOutAll` together with `fadeOutNonPersistentRipples`. Alternatively we could expose `_activeRipples` from the renderer and deal with it explicitly from within `MatRipple`.

Reply:
Got it. I've created another commit and update the changes